python/ray/llm/_internal/serve/deployments/llm/vllm/vllm_engine.py

```diff
@@ -816,9 +816,9 @@ async def check_health(self) -> None:
             raise RuntimeError(f"{type(self.engine)} does not support health check.")
 
         try:
-            return await asyncio.wait_for(self.engine.check_health(), timeout=15)
+            await asyncio.wait_for(self.engine.check_health(), timeout=15)
```

Comment:
removed the timeout time since ray serve has an adjustable timeout per deployment anyways.